test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -0,0 +1,1675 @@
+        value = (QSTAT_QF_PATH, [QSTAT_CMD, "-Qf"])
+        self.server_info[QSTAT_QF_OUT] = value
+        value = (QMGR_PR_PATH, [QMGR_CMD, "-c", "p r"])
+        self.server_info[QMGR_PR_OUT] = value
+        value = (RSCS_PATH, None)
+        self.server_info[RSCS_ALL] = value
+        value = (SVR_PRIV_PATH, None)
+        self.server_info[SVR_PRIV] = value
+        value = (SVR_LOGS_PATH, None)
+        self.server_info[SVR_LOGS] = value
+        value = (ACCT_LOGS_PATH, None)
+        self.server_info[ACCT_LOGS] = value
+
+        # Job information
+        value = (QSTAT_PATH, [QSTAT_CMD])
+        self.job_info[QSTAT_OUT] = value
+        value = (QSTAT_F_PATH, [QSTAT_CMD, "-f"])
+        self.job_info[QSTAT_F_OUT] = value
+        value = (QSTAT_T_PATH, [QSTAT_CMD, "-t"])
+        self.job_info[QSTAT_T_OUT] = value
+        value = (QSTAT_TF_PATH, [QSTAT_CMD, "-tf"])
+        self.job_info[QSTAT_TF_OUT] = value
+        value = (QSTAT_X_PATH, [QSTAT_CMD, "-x"])
+        self.job_info[QSTAT_X_OUT] = value
+        value = (QSTAT_XF_PATH, [QSTAT_CMD, "-xf"])
+        self.job_info[QSTAT_XF_OUT] = value
+        value = (QSTAT_NS_PATH, [QSTAT_CMD, "-ns"])
+        self.job_info[QSTAT_NS_OUT] = value
+        value = (QSTAT_FX_DSV_PATH, [QSTAT_CMD, "-fx", "-F", "dsv"])
+        self.job_info[QSTAT_FX_DSV_OUT] = value
+        value = (QSTAT_F_DSV_PATH, [QSTAT_CMD, "-f", "-F", "dsv"])
+        self.job_info[QSTAT_F_DSV_OUT] = value
+
+        # Node/MoM information
+        value = (PBSNODES_VA_PATH, [PBSNODES_CMD, "-va"])
+        self.node_info[PBSNODES_VA_OUT] = value
+        value = (PBSNODES_A_PATH, [PBSNODES_CMD, "-a"])
+        self.node_info[PBSNODES_A_OUT] = value
+        value = (PBSNODES_AVSJ_PATH, [PBSNODES_CMD, "-avSj"])
+        self.node_info[PBSNODES_AVSJ_OUT] = value
+        value = (PBSNODES_ASJ_PATH, [PBSNODES_CMD, "-aSj"])
+        self.node_info[PBSNODES_ASJ_OUT] = value
+        value = (PBSNODES_AVS_PATH, [PBSNODES_CMD, "-avS"])
+        self.node_info[PBSNODES_AVS_OUT] = value
+        value = (PBSNODES_AS_PATH, [PBSNODES_CMD, "-aS"])
+        self.node_info[PBSNODES_AS_OUT] = value
+        value = (PBSNODES_AFDSV_PATH, [PBSNODES_CMD, "-aFdsv"])
+        self.node_info[PBSNODES_AFDSV_OUT] = value
+        value = (PBSNODES_AVFDSV_PATH, [PBSNODES_CMD, "-avFdsv"])
+        self.node_info[PBSNODES_AVFDSV_OUT] = value
+        value = (QMGR_PN_PATH, [QMGR_CMD, "-c", "p n @default"])
+        self.node_info[QMGR_PN_OUT] = value
+        value = (MOM_PRIV_PATH, None)
+        self.node_info[MOM_PRIV] = value
+        value = (MOM_LOGS_PATH, None)
+        self.node_info[MOM_LOGS] = value
+
+        # Comm information
+        value = (COMM_LOGS_PATH, None)
+        self.comm_info[COMM_LOGS] = value
+
+        # Hook information
+        value = (QMGR_PH_PATH, [QMGR_CMD, "-c", "p h @default"])
+        self.hook_info[QMGR_PH_OUT] = value
+        value = (QMGR_LPBSHOOK_PATH, [QMGR_CMD, "-c", "l pbshook"])
```

Comment:
I just followed Scott's suggestions on the forum so I'm guessing here: pbshook is a built-in hook, there's no need to create it, so there's actually no print output generated for it by qmgr. For site hooks, it might be useful to get the print output as one may want to create the same hook on a test system for debugging.
If you aren't satisfied with this answer then I'd ask that you raise this question on the forum so that the Scott and others can provide further clarification.